### PR TITLE
Feature/add time modal component

### LINKED ISSUE #63: JSX/CSS: Build out AddTime component
- [ ] Build out AddTime component
- [ ] Format and style AddContact modal display
- [ ] Add End Trip functionality / page routing
- [ ] Add Extend Time functionality / page routing

### PR DESCRIPTION
**Pull request by pcmueller**
## Type of Change
- [x] JSX
- [ ] HTML
- [x] JavaScript
- [x] CSS
- [x] React
- [x] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
AddTime component

## Please give a short description of the changes that were made below

- Feature:
- Built out `AddTime` component functionality and UI
- Added an `extendedTimeOptions` dataset to assets, which is used to populate the dropdown menu.
- Connected user options to proper page routing - EndTrip reroutes to MainView display, selection of Extend Time dropdown options closes modal and refreshes TripView display, with conditional rendering to restart trip timers with selected amount of time.
- Added functionality to TripETA component to take ETA integer and reduce it into hours and minutes, when applicable, to be displayed in string form.  
- Set us up to connect sad path functionality - left a spot on line 20 of `AddTime` to invoke ALERT page routing / SMS messaging.

## How has this been tested?
- [x] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/AddTime/AddTime.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import ReactModal from 'react-modal';
+import { CountdownCircleTimer } from 'react-countdown-circle-timer';
+import { NavLink } from 'react-router-dom';
+import extendedTimeOptions from '../../assets/extendedTimeOptions';
+import addTimeModalStyles from './jsxStyles/addTimeModalStyles';
+import addTimeDropdownStyles from './jsxStyles/dropdownStyles';
+
+ReactModal.setAppElement('#root');
+
+function AddTime( { setExtension, setEtaSeconds, modalIsOpen, closeModal } ) {
```
Comment (bhohnco) [inline src/Components/AddTime/AddTime.js:12]: This looks awesome. We'll have to brainstorm how we wait to hook up the end trip / extend the trip option for the message.

File: src/Components/AddTime/jsxStyles/dropdownStyles.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  }),
+  input: () => ({
+    color: 'transparent',
+  }),
+  placeholder: (defaultStyles) => {
+    return {
+      ...defaultStyles,
+      alignSelf: 'center',
+      justifySelf: 'center',
+      fontSize: '1.5em',
```
Comment (bhohnco) [inline src/Components/AddTime/jsxStyles/dropdownStyles.js:15]: Solid work Pete!

File: src/Components/CurrentTrip/CurrentTrip.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                setSecondsActive(etaSeconds - totalElapsedTime > 0);
+                return [etaSeconds - totalElapsedTime > 0]
+              }}
+            >
+              {({ elapsedTime }) => renderTime('seconds', getTimeSeconds(elapsedTime))}
+            </CountdownCircleTimer>
+          </article>
+        }
+        <NavLink exact to='/'>
+          <button onClick={endTrip} className='end-walk-btn'>
```
Comment (bhohnco) [inline src/Components/CurrentTrip/CurrentTrip.js:148]: easy to read and clear, nice job

File: src/Components/TripETA/jsxStyles/etaModalStyles.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const etaModalStyles = {
+  content: {
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    height: '40%',
+    width: '45%',
```
Comment (bhohnco) [inline src/Components/TripETA/jsxStyles/etaModalStyles.js:8]: pumped to see how this looks

File: src/Components/AddTime/AddTime.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        <section className='add-time-response'>
+          <NavLink exact to='/'>
+            <button onClick={closeModal} className='end-trip-btn'>
+              END TRIP
+            </button>
+          </NavLink>
+          <Select
+            className='extend-time'
+            placeholder='EXTEND TIME'
+            styles={addTimeDropdownStyles}
```
Comment (carolineshowscode) [inline src/Components/AddTime/AddTime.js:89]: Haven't seen jsx imported like this. Pretty cool idea.

**Review by carolineshowscode — APPROVED** [review]
Extension idea - we could add in a 4th text message option here where we send another SMS alert with an updated ETA should the user opt to add more time to their trip when they are prompted by the modal

**Review by bhohnco — APPROVED** [review]

**Pull request merged**